addons/mail/static/src/discuss/call/common/thread_model_patch.js

```diff
@@ -65,11 +72,20 @@ const ThreadPatch = {
                 }
             },
         });
-    },
-    get videoCount() {
-        return Object.values(this.store["discuss.channel.rtc.session"].records).filter(
-            (session) => session.hasVideo
-        ).length;
+        this.videoCount = fields.Attr(0, {
+            compute() {
+                return this.rtc_session_ids.filter((s) => s.hasVideo).length;
+            },
+            onUpdate() {
+                if (this.fullscreenIndicator === INDICATOR.DISCARDED) {
+                    return;
+                }
+                this.fullscreenIndicator =
+                    this.videoCount > 0 && this.chat_window?.isOpen
+                        ? INDICATOR.ACTIVE
+                        : INDICATOR.DISABLED;
+            },
```

Comment:
It should not pulse when the transition comes from self-enabled video.
The other uses of `videoCount` is about whether one of the card show a video, so the current definition of `videoCount` is good for them. For `fullscreenIndicator`, it should be another flavor of `videoCount` that filters-out self videos.